src/main/java/backend/RepoIO.java

```diff
@@ -207,6 +207,10 @@ private Model downloadRepoFromSourceBlocking(String repoId) {
         return repoSource.replaceIssueMilestone(issue, milestone);
     }
 
+    public CompletableFuture<Boolean> editIssueState(TurboIssue issue, boolean open) {
```

Comment:
Boolean variables should be `isOpen`. This should apply to call instances of `open` boolean parameter or variable in other parts.